internal/docstore/driver/driver.go

```diff
@@ -127,3 +127,7 @@ type DocumentIterator interface {
 	// needed.
 	Stop()
 }
+
+// EqualOp is the name of the equality operator.
+// It is defined here to avoid confusion between "=" and "==".
+const EqualOp = "="
```

Comment:
Should this go into the portable API so that the user uses this as well? I'm not sure whether all drivers have the same standard of using "=".

Reply:
We should pick one as the docstore standard, and `=` is a slightly better choice because it matches SQL.

If a user picks the wrong one they'll get an error and fix their code. I added this because I'm worried that a driver will get it wrong, and that would be a real bug.

Let's leave it here and we can always "promote" it later if we think that's a good idea.